.concourse/tasks/upgrade.sh

```diff
@@ -15,16 +15,19 @@ export FORCE_DELETE=true
 export HELM_VERSION="v3.1.1"
 export BACKEND=imported
 export QUIET_OUTPUT=true
-export GKE_CLUSTER_NAME="kubecf-ci-$(cat semver.gke-cluster/version | sed 's/\./-/g')"
-export KUBECFG="$(readlink -f ~/.kube/config)"
+GKE_CLUSTER_NAME="kubecf-ci-$(sed 's/\./-/g' < "semver.gke-cluster/version")"
+export GKE_CLUSTER_NAME
+KUBECFG="$(readlink -f ~/.kube/config)"
+export KUBECFG
 
-printf "%s" '((gke-suse-cap-json))' > $PWD/gke-key.json
+printf "%s" '((gke-suse-cap-json))' > "${PWD}/gke-key.json"
 export GKE_CRED_JSON=$PWD/gke-key.json
-gcloud auth activate-service-account --key-file $PWD/gke-key.json
+gcloud auth activate-service-account --key-file "${PWD}/gke-key.json"
 
-export GKE_PROJECT={{ if has . "gke_project" }}{{ .gke_project }}{{ else }}"suse-225215"{{ end }}
-export GKE_ZONE={{ if has . "gke_zone" }}{{ .gke_zone }}{{ else }}"europe-west3-c"{{ end }}
-export GKE_CLUSTER_NAME="kubecf-ci-$(cat semver.gke-cluster/version)"
+export GKE_PROJECT='{{ if has . "gke_project" }}{{ .gke_project }}{{ else }}{{ "suse-225215" }}{{ end }}'
+export GKE_ZONE='{{ if has . "gke_zone" }}{{ .gke_zone }}{{ else }}{{ "europe-west3-c" }}{{ end }}'
```

Comment:
```suggestion
export GKE_PROJECT='{{ .gke_project | default "suse-225215" }}'
export GKE_ZONE='{{ .gke_zone | default "europe-west3-c"}}'
```
Confirmed with [this](https://play.golang.org/p/CXSMe_6wmwI) that we can access non-existent keys and it will return a `nil` instead of erroring out immediately.  I believe this is run via gomplate, which has a [`default`](https://docs.gomplate.ca/functions/conv/#conv-default).

Reply:
The trick is in `if has` (https://docs.gomplate.ca/functions/conv/#conv-has-deprecated). ```Can be used with if to prevent the template from trying to access a non-existent property in an object.
```. I don't remember if we ever tried the `| default` syntax. Looks simpler, let's keep it.